node/src/BaseClient.ts

```diff
@@ -306,17 +314,13 @@ export type DecoderOption = {
 export function convertFieldsAndValuesForHset(
```

Comment:
```suggestion
export function convertFieldsAndValues(
```
Can you rename please?
Shouldn't we also move to commands.ts?